config/dev.exs

```diff
@@ -9,10 +9,10 @@ import Config
 config :hello_world, HelloWorldWeb.Endpoint,
   # Binding to loopback ipv4 address prevents access from other machines.
   # Change to `ip: {0, 0, 0, 0}` to allow access from other machines.
-  http: [ip: {127, 0, 0, 1}, port: 4000],
+  http: [ip: {127, 0, 0, 1}, port: String.to_integer(System.get_env("PORT") || "6060")],
   check_origin: false,
   code_reloader: true,
-  debug_errors: true,
+  debug_errors: false,
```

Comment:
@manusajith is it recommended to set this to false for development?

Reply:
@sestrella I changed this to false, so that when the user hit 404(invalid) routes, it returns the the response in the format that we expect(excluding the debug information)

Reference: https://hexdocs.pm/phoenix/custom_error_pages.html#the-errorview

Reply:
Oh kind of like `consider_all_requests_local` in the context of Rails, got it 👍🏼

Reply:
ryt